docs/building.md

```diff
@@ -22,6 +22,11 @@ Ensure this section works for you before jumping to the Android/iOS specific sec
   - zlib
     - Using apt: `apt install zlib1g-dev`
     - Already installed on macOS by XCode.
+  - uniffi-bindgen
+    - `cargo install uniffi_bindgen`. Note that this is being updated regularly
+      and cargo isn't very clever - if you see complaints about the wrong
+      version being installed, you probably need to
+      `cargo uninstall uniffi_bindgen` and re-execute the installation command.
```

Comment:
Thanks for adding this!

IIUC `cargo install -f uniffi_bindgen` should also work for upgrading to the latest version. Also this is mostly about `uniffi-bindgen` being not very clever, not cargo ;-)